ui/src/alerting/components/ChecksColumn.tsx

```diff
@@ -23,13 +26,46 @@ const ChecksColumn: FunctionComponent<Props> = ({
   checks,
   router,
   params: {orgID},
+  rules,
+  endpoints,
 }) => {
   const handleClick = () => {
     router.push(`/orgs/${orgID}/alerting/checks/new`)
   }
+
+  const tooltipContents = (
+    <>
+      A <strong>Check</strong> is a periodic query that the system
+      <br />
```

Comment:
Do these `<br />` have to be here? Feels like an indication that something else could be fixed

Reply:
I manually placed the line breaks in a way that makes it nice to read rather than relying on left/right padding